api/seeds/development/users.js

```diff
@@ -3,7 +3,6 @@ const bcrypt = require('bcryptjs');
 exports.seed = async knex => {
   await knex('users').insert([
     {
-      id: 57,
```

Comment:
Back in the old days, before you could log in, we hard-coded the user ID as `57` somewhere in the app.  Something to do with ketchup.  Accordingly, we made the development user have ID 47.  Now that you can actually log in and we don't have to synchronize the hard-coded values, let's un-hard-code this one.

Reply:
Now I want some Heinz 57 sauce.